Shared/Extensions/URLExtensions.swift

```diff
@@ -238,19 +238,28 @@ extension URL {
     }
 
     /**
-     * Returns just the domain, but with the same scheme, and a trailing '/'.
```

Comment:
This comment is actually outdated, see https://github.com/brave/brave-ios/commit/0158c51c503a151facbab2b8b0d01950b6096050#diff-005e296c633487f7dcebd610aa8bdd11